Formula/box2d.rb

```diff
@@ -38,7 +40,8 @@ def install
     system "cmake", ".", *args
     system "cmake", "--build", "."
     system "cmake", "--install", "."
-    pkgshare.install "unit-test/hello_world.cpp", "unit-test/doctest.h"
+    pkgshare.install "unit-test/hello_world.cpp"
+    pkgshare.install_symlink Formula["doctest"].opt_include/"doctest/doctest.h"
```

Comment:
The symlink created this way will use a `Cellar` path, so it will break when `doctest` gets a version/revision bump.

Also, this will be a broken symlink during the test.

Reply:
Hmm, I thought `opt_include` was the way to avoid a Cellar path.  Should we use `ln_sf` here instead?

Reply:
`#install_symlink` calls `realpath`. https://github.com/Homebrew/brew/blob/7a14ae61874770d9ceaee716f90e19c3b5674caa/Library/Homebrew/extend/pathname.rb#L163

You can use `ln_s`. Though I would just not create this symlink at all and pass an `-I` flag in the `test`. We can `inreplace "hello_world.cpp"` if it has the wrong `#include` directive for that.

Reply:
The `-I` flag to the `doctest` include dir was all that was needed.  Much better than symlinking!